instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/builder/internal/DefaultHttpClientInstrumenterBuilder.java

```diff
@@ -225,6 +240,10 @@ public Instrumenter<REQUEST, RESPONSE> build() {
           .addAttributesExtractor(HttpExperimentalAttributesExtractor.create(attributesGetter))
           .addOperationMetrics(HttpClientExperimentalMetrics.get());
     }
+    if (redactSensitiveUrlParameters) {
+      builder.addAttributesExtractor(
+          HttpClientExperimentalHttpParamsRedactionExtractor.create(attributesGetter));
+    }
```

Comment:
so the idea is that you overwrite the attribute set in `HttpClientAttributesExtractor`? why not just modify the `HttpClientAttributesExtractor`?